### ISSUE TITLE
[Bug]: 2024.1.1 is packaged with Python 3.8.x but will not run locally with said version

### ISSUE THREAD

**Issue by boazrl** [psychopy/psychopy#6361]
### PsychoPy Version

2024.1.1

### What OS are your PsychoPy running on?

Windows 11

### Bug Description

the failure results from versionchooser.py having this map on line 32 
```
# define ranges of PsychoPy versions which support each Python version
versionMap = OrderedDict({
    Version('2.7'): (Version("0.0"), Version("2020.2.0")),
    Version('3.6'): (Version("1.9"), Version("2022.1.0")),
    Version('3.8'): (Version("2022.1.0"), Version("2024.1.0")),
    Version('3.10'): (Version("2023.2.0"), None),
})
```

should you raise the maximum version of 3.8 in the versionMap, or pack the standalone installer with python 3.10?
otherwise the standalone is not valid. 

### Expected Behaviour

Psychopy standalone installation should work without the need for manipulating environments \ having an external python installation. 

### Steps to Reproduce

1. install 2024.1.1 from the standalone installer. 
2. run your experiment locally 
3. fail

**Comment by krishna-raj-sr** [comment]
Same issue here with the standalone installer. Were you able to find any solution?

**Comment by peircej** [comment]
Yes, apologies, we have seen this and it will be fixed for 2024.1.2 but, in the meantime, if you set your useVersion setting to the blank entry then it will not make this check and use the currently-installed version instead so that should work
